content/posts/buying_domain.txt

```diff
@@ -0,0 +1,43 @@
+
+# Buying your first domain
+
+As a brand new company, you're going to need a domain name rapidly. This domain name will quickly be used for your website,  email, and generic "proof of existence".
+
+It'll become the root for the identity related to your company.
+
+Why would you need a website rapidly, if you have nothing ready to sell yet? Many services require being linked to a domain. For example, to get onboarded to Apple's Device Enrolment Program, which lets you drop-ship laptops to employees directly from Apple but ready to go with your configuration, you need to have an HTTPS website. To get a certificate, you must be able to prove you control a domain. Therefore, there's no time to waste, you should get your domain as fast as possible when starting a new company.
+
+## Picking a domain name
+
+1. Ensure your company name and the domain you want to buy are available and legally appropriate. I am not a lawyer and don't pretend to be one on the Internet, but I recommend using real ones.
+2. Avoid "weird" [top-level domains](https://www.zdnet.com/article/new-top-level-domains-a-money-grab-and-a-mistake-paul-vixie/). You do not want your company to depend on a small country's control of a TLD, and you definitely don't want it to use a domain name known for being used for [a lot of malicious activity](https://www.farsightsecurity.com/assets/media/download/VB2018-study.pdf). Pick an old favorite such as .com, even if it means not getting the perfect short name (you can get the weird TLD ones as well, but make your primary the "good" one.)
+
+## Buying the domain name
+
+Once you’ve picked your domain, you need to buy it. If you’re a startup, you don’t have any infrastructure yet, except for personal email accounts.
+
+
+0. Ensure you have Multi-Factor Authentication enabled on your account, ideally, using a method that is not SMS or voice based. Your email account will be used to buy the domain, and until you have corporate email set up, could turn out to be the weakest link towards attacking your new company. It must be as safe as possible
+1. Pick  a registrar that supports Multi-Factor Authentication. Many do by now, though [some](https://www.hover.com) don’t [murder elephants](https://science.time.com/2011/04/04/godaddy-ceo-on-shooting-an-elephant-im-not-sorry/), and enable MFA as fast as possible.
+2. Buy the domain for multiple years. You want to be sure you have time to move to a more corporate type account by the time it gets renewed.
+3. Configure [domain privacy](https://en.wikipedia.org/wiki/Domain_privacy). While it looks shady for a company to hide details from [WHOIS](https://en.wikipedia.org/wiki/WHOIS), you don’t want to share your information until you have a real corporate address and email account.
+3. Configure the [Registrar Lock](https://www.icann.org/resources/pages/locked-2013-05-03-en). This will ensure your domain can’t easily be transferred to another registrar out of your control.
```

Comment:
You've got two 3s in this numbered list. 

Also, what about purchasing name variations likely to cause confusion or get typosquatted? Do you want to pick up .com, .org, .net and .io right out of the gate, or do you wait until later to do that?

Reply:
Re the numbered lists: Markdown takes care of it, you could number them all 1. and it would do it right